cocos/game/splash-screen.ts

```diff
@@ -536,7 +549,10 @@ export class SplashScreen {
                     );
 
                     cmdBuff.bindPipelineState(bgPso);
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                    cmdBuff.bindDescriptorSet(SetIndex.GLOBAL, pipeline.descriptorSet);
                     cmdBuff.bindDescriptorSet(SetIndex.MATERIAL, bgPass.descriptorSet);
+                    cmdBuff.bindDescriptorSet(SetIndex.LOCAL, this.localBGDesc);
```

Comment:
Why GLOBAL, LOCAL descriptorSets are needed here?
Are they also needed in other backends?

Reply:
I have the same question.

Reply:
The splash-screen shader only defines a material corresponding to setindex 1, but WebGPU performs strict checks on this item, which means that you cannot skip setindex 0 and only bind setindex 1. The binding of setindex corresponding to local can be removed, and the global bindgroup needs to be bound, even though global has no data.